orttraining/orttraining/training_api/include/onnxruntime_training_c_api.h

```diff
@@ -296,6 +296,23 @@ struct OrtTrainingApi {
   *
   */
   ORT_CLASS_RELEASE(CheckpointState);
+
+  /** \brief Export a model that can be used for inferencing.
+  *
+  * If the training session was provided with an eval model, the training session can generate
+  * an inference model if it knows the inference graph outputs. The inference graph outputs are expected
+  * to be set in the metadata_props field at the time of eval model creation. This happens by default
+  * using onnxruntime offline tooling.
```

Comment:
from your PR description, this also happens on device, right?

Reply:
The export to inference happens on device. The metadata prop field is populated during offline tooling step.